src/components/user-list/mocks.ts

```diff
@@ -0,0 +1,78 @@
+export const users = [
```

Comment:
constants should be uppercase

Reply:
it's mocked data, I will delete it after backend is ready

Reply:
Fixed colors